modules/phase_field/src/kernels/CHParsed.C

```diff
@@ -54,13 +56,13 @@ CHParsed::computeGradDFDCons(PFFunctionType type)
   {
     case Residual:
       for (unsigned int i = 0; i <= _nvar; ++i)
-        res += (*_grad_vars[i])[_qp] * (*_second_derivatives[i])[_qp];
+        res += (*_grad_vars[i])[_qp]*(*_second_derivatives[i])[_qp];
       return res;
 
     case Jacobian:
-      res = _grad_phi[_j][_qp] * (*_second_derivatives[0])[_qp];
+      res = _grad_phi[_j][_qp]*(*_second_derivatives[0])[_qp];
       for (unsigned int i = 0; i <= _nvar; ++i)
-        res += _phi[_j][_qp] * (*_grad_vars[i])[_qp] * (*_third_derivatives[i])[_qp];
+        res += _phi[_j][_qp]*(*_grad_vars[i])[_qp]*(*_third_derivatives[i])[_qp];
```

Comment:
There are several places in this PR where you collapse the space around the binary operators. In general we encourage you to leave space around operators. It's part of the code standard and it helps with readability. Other than that I think this PR looks fine.